### PR TITLE
Use postman-echo to replace httpbin

### PR DESCRIPTION
**Pull request by rentziass**
httpbin timeouts are causing our http-client tests to timeout too, this changes httpbin to postman-echo (Postman now also owns httpbin).

### CODE REVIEW

**Comment by wsiim** [comment]
في ثلاثاء، 23 أيار، 2023 في 2:33 م، كتب Francesco Renzi <
***@***.***>:

File: packages/http-client/__tests__/basics.test.ts [hunk 1]
```diff
@@ -311,63 +313,63 @@ describe('basics', () => {
...
-    expect(restRes.result?.url).toBe('https://httpbin.org/put')
+    expect(restRes.result?.url).toBe('https://postman-echo.com/put')
     expect(restRes.result?.json.name).toBe('foo')
 
-    expect(restRes.result?.headers['Accept']).toBe(
+    expect(restRes.result?.headers[httpm.Headers.Accept]).toBe(
       httpm.MediaTypes.ApplicationJson
     )
-    expect(restRes.result?.headers['Content-Type']).toBe(
+    expect(restRes.result?.headers['content-type']).toBe(
```
Comment (Link-) [inline packages/http-client/__tests__/basics.test.ts:348]: nit: isn't the header key `Content-Type`? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type 

Is postman returning it lower case?

**Review by Link- — APPROVED** [review]
🛳️ ship!

**Review by Liabaemt — COMMENTED** [review]
Fix

**Pull request merged**